src/mars.c

```diff
@@ -1193,6 +1194,7 @@ Language changes listed by -transition=id:\n\
         VersionCondition::addPredefinedGlobalIdent("D_NoBoundsChecks");
 
     VersionCondition::addPredefinedGlobalIdent("D_HardFloat");
+    objc_tryMain_dObjc();
```

Comment:
[That `objc_tryMain_dObjc` function][1] just adds a predefined version identifier in case you want to use `version (D_ObjectiveC)` to detect whether Objective-C features are available at compile-time. (If your backend does not support emitting Objective-C code, then you probably shouldn't add the 'D_ObjectiveC' version identifier.)

 [1]: https://github.com/D-Programming-Language/dmd/pull/4321/files#diff-b89e41d4e594bcc155de0109a27257e5R145